src/test/java/test/LoginTest.java

```diff
@@ -0,0 +1,46 @@
+package test;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.WebDriver;
+import pages.AccountModel;
+import pages.LoginPage;
+import pages.ProfilePage;
+import pages.WebDriverSingletion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(Listener.class)
+
+public class LoginTest {
+    private LoginPage loginPage;
+    private ProfilePage profilePage;
+    private AccountModel accountModel;
+    private WebDriver driver;
+
+    @BeforeAll
+    public void setUp() {
+        driver = WebDriverSingletion.getInstance().openBrowser();
+        loginPage = new LoginPage();
+        accountModel = new AccountModel();
```

Comment:
This line is incorrect, remove it. You are not on this page to create page object instance